assets/css/rop.css

```diff
@@ -4348,3 +4348,21 @@
 .rop-post-type-badge a {
 	text-decoration: underline;
 }
+
```

Comment:
Please run `grunt local ` command on your fork before committing so the code formatting can be in line with ours